packages/flutter_tools/lib/src/localizations/message_parser.dart

```diff
@@ -0,0 +1,519 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Symbol Types
+import 'gen_l10n_types.dart';
+
+enum ST {
+  // Terminal Types
+  openBrace,
+  closeBrace,
+  comma,
+  equalSign,
+  other,
+  plural,
+  select,
+  string,
+  number,
+  identifier,
+  empty,
+  // Nonterminal Types
+  message,
+
+  placeholderExpr,
+
+  pluralExpr,
+  pluralParts,
+  pluralPart,
+
+  selectExpr,
+  selectParts,
+  selectPart,
+}
+
+final List<ST> nonterminals = <ST>[
```

Comment:
Doesn't looks like this is referenced, can we delete? If not, let's make it `const`.